satpy/readers/aapp_l1b.py

```diff
@@ -33,10 +33,22 @@
 import xarray as xr
 from dask import delayed
 
-from satpy import CHUNK_SIZE
 from satpy.readers.file_handlers import BaseFileHandler
+from satpy.utils import get_chunk_pixel_size
+
+CHANNEL_DTYPE = float
+
+
+def get_avhrr_lac_chunks(shape, dtype):
+    """Get chunks from a given shape adapted for full-resolution AVHRR data."""
+    limit = get_chunk_pixel_size() * np.dtype(dtype).itemsize
```

Comment:
Why can't the dtype logic be put in the `get_chunk_pixel_size` function?

Reply:
yes, it can, but then we have to change the name of that function. On it.

Reply:
Well is `pixel_size` even the right descriptor here? It isn't the chunk size of a pixel? It is the chunk size in pixels, right? So `get_chunk_size_in_pixels`? Or maybe more generically in not say `pixels` at all since chunk size is always based on elements (pixels).

So for dtype it would be `get_chunk_size_for_dtype`?